wikimedia/wikipedia-ios
# Remove unnecessary components

**Phabricator:** 
https://phabricator.wikimedia.org/T374163

### Notes
* 

### Test Steps
Open app (logged in)
Navigate to settings
Check to make sure "Donate" is removed, if logged in, also check that the account view has removed unnecessary components
Repeat steps with logged out
Build number:
168

### Screenshots/Videos
![Simulator Screenshot - iPhone 15 Pro - 2024-09-20 at 16 28 07](https://github.com/user-attachments/assets/6f9b39c3-1863-4ee4-b68d-f604eefc03c1)
![Simulator Screenshot - iPhone 15 Pro - 2024-09-20 at 16 28 11](https://github.com/user-attachments/assets/63078af8-d552-49a1-819d-703f70483739)
![Simulator Screenshot - iPhone SE (3rd generation) - 2024-09-20 at 16 29 41](https://github.com/user-attachments/assets/2431bc46-2d60-468d-9ee5-7f1772bb9d31)
